pkg/util/provider/machinecontroller/machine_util.go

```diff
@@ -911,7 +911,7 @@ func (c *controller) drainNode(deleteMachineRequest *driver.DeleteMachineRequest
 
 			drainOptions := drain.NewDrainOptions(
 				c.targetCoreClient,
-				timeOutDuration,
+				timeOut,
```

Comment:
Is it that MCM restarts can reset the drainTimeout? I hope there are no side-effects from it.
Does this also need to be fixed in-tree then? https://github.com/gardener/machine-controller-manager/blob/master/pkg/controller/machine.go#L708 ?

Reply:
Oops. Nice catch. Wait if it's working as expected with in-tree why the difference in OOT. I guess the issue is something else then. Let me re-check.